### DISCUSSION TITLE
sbt 1.5.0 timing

### DISCUSSION THREAD

**Discussion by eed3si9n** [sbt/sbt#6342] [General]
When should sbt 1.5.0 start the RC cycle? It usually takes a few rounds (1~2 week each). Hopefully things are a bit more stable since we've been pushing out the milestones.

cc @smarter, @adpi2

**Comment by adpi2** [comment]
I propose the following path going forward:

- [x] I create a branch in the dotty repo and in the dotty community build that uses sbt 1.5.0-M2 everywhere, instead of sbt-dotty, and pass the CI
- [x] We get the approval from the dotty team that the naming and versioning of the Scala 3 artifacts will not change
It applies specifically to `scala3-library`, `scala3-compiler` and `scaladoc`, but also to `scala3-interfaces`, `tasty-core` and `scala3-tasty-inspector`, to a lesser extent.
- [ ] We start the RC cycle

**Reply by eed3si9n** [reply]
Sounds good to me.

**Reply by adpi2** [reply]
Making some progress in: https://github.com/lampepfl/dotty/pull/11568

**Reply by adpi2** [reply]
The CI is passing in lampepfl/dotty#11568 which shows that sbt 1.5.0 can replace sbt-dotty in the dotty build, in the scripted tests and the community projects.

Also I got the approval from the dotty team (during the LAMP meeting) that:
- the artifact names will not change
- the Scala binary version will be "3"
- sbt-dotty will be deprecated in favor of sbt 1.5.0

So we are good to start the RC cycle.
